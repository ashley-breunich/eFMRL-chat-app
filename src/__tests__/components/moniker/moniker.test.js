import React from 'react';
import renderer from 'react-test-renderer';
import Moniker from 'components/moniker/moniker.js';

describe('<Header />', () => {
  it('is alive at application start', () => {
    let app = shallow(<Moniker />);
    expect(app.find('form').exists()).toBeTruthy();
  });

  it('is alive at application start', () => {
    let app = shallow(<Moniker />);
    expect(app.find('input').exists()).toBeTruthy();
  });

  it('is alive at application start', () => {
    let app = shallow(<Moniker />);
    expect(app.find('p').exists()).toBeTruthy();
  });

  it('renders correctly', () => {
    const header = renderer.create(<Moniker />).toJSON();
    expect(header).toMatchSnapshot();
  });
});
