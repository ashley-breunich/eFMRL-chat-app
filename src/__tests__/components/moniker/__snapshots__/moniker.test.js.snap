// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders correctly 1`] = `
<form
  className="loginForm"
>
  <input
    className="loginInput"
    name="typedInput"
    placeholder="What is your name?"
  />
  <p
    className="loginError"
  />
</form>
`;
